lib/clientv2/prefixdeletewatch_e2e_test.go

```diff
@@ -1,84 +0,0 @@
-// Copyright (c) 2016-2017 Tigera, Inc. All rights reserved.
```

Comment:
This file is now watch_e2e_test.go.  The first test is tiger same prefix delete test, but the second test is a multi-threaded events/watch/cancel test.